### ISSUE TITLE
[Suggestion] Improve Fast Left Click controls

### ISSUE THREAD

**Issue by 1schlick** [maruohon/tweakeroo#386]
Instead of having to go into the GUI to set the number of clicks per tick, you should only have to scroll while holding the toggle button to set the value. Same goes for fast right click.

**Comment by maruohon** [comment]
Indeed, and this was added 1.5 years ago in the main development code in 1.12.2, here: c043cd52a071f4898f24ec1f079a04bb1d04a4f5
